deployer/aws-lambda/content-origin-request/build.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable node/no-unpublished-require */
```

Comment:
Later we could reuse this mapping to support these redirects with MDN Offline. 🤔